python/cudf/cudf/tests/test_string.py

```diff
@@ -3423,3 +3423,44 @@ def test_str_join_lists(sr, sep, string_na_rep, sep_na_rep, expected):
         sep=sep, string_na_rep=string_na_rep, sep_na_rep=sep_na_rep
     )
     assert_eq(actual, expected)
+
+
+def test_str_find_multiple():
+    s = cudf.Series(["strings", "to", "search", "in"])
+    t = cudf.Series(["a", "s", "g", "i", "o", "r"])
```

Comment:
Similarly to above, I'd like some strings that are not single characters.

Reply:
Done